### ISSUE TITLE
include 'about' into section heading

### ISSUE THREAD

**Issue by werkstattcodes** [siegerts/hugo-theme-basic#9]
Hi,

first of all many thanks for this very elegant theme! 

I am struggling to adapt the theme so that the 'heading row' reads as 'Hugo Theme Basic / About ' when clicking on the 'about' link.  Currently, the heading remains unaltered and 'only' reads as 'Hugo Theme Basic'.  However, when I click on 'post series' the heading reads as 'Hugo Theme Basic / Series'. So I wanted some consistency here and use the heading row as something like a breadcrumbtrail.

I included 'about' into the taxomonies in the config.toml what results in the heading 'Hugo Theme Basic / About".  However, I am struggling for days now to make the text of the about.md file (in the content folder) appear under the modified heading. Any help? Many thanks. 

fyi - here my clumsy attempts [link](https://github.com/zoowalk/blog).

**Comment by siegerts** [comment]
Hi @zoowalk !  I'm happy that you are finding the theme useful!

I made a few changes to display the `<site title> / About` breadcrumbs show on your `/about` page.

First, I changed the `about.md` type to `page` in the front matter config. This will give it the the _about.md_ page the properties of the base _page_ archetype (https://gohugo.io/templates/single-page-templates/).

Then, I added a theme override for the way that single `page`s are displayed.  The _theme overide_ allows you to make your own custom changes on top of the base theme(s) that you are using.  And, if for some reason the base theme changes and your project `git pulls` then your custom changes will remain since they live in your project and not the theme submodule.

Hopefully this helps!

https://github.com/zoowalk/blog/pull/1
